cognite/client/data_classes/raw.py

```diff
@@ -24,7 +24,7 @@ def __init__(
         self._cognite_client = cognite_client
 
     # GenStop
-    def to_pandas(self):
+    def to_pandas(self, **kwargs):
```

Comment:
Should really try to avoid using kwargs here. It makes for some unexpected behaviour on the users part. They will pass in arguments that are never used and not understand why they don't have an effect, I've seen it many times before.

Why do you need it?

Reply:
These derived objects implement to_pandas in a way which  already accepted fewer arguments than the base class before the camel case change (ignore, expand).  Adding some parameters explicitly with docstring 'this is ignored' didn't seem right either.

Reply:
Is it because these methods are called from the superclass' _repr_html_ method with the camel_case argument?

Reply:
yep, that's why they all need to accept camel_case.